PlainTasks (Linux).sublime-settings

```diff
@@ -1,4 +1,4 @@
-{
+bl PL{
```

Comment:
I have no idea. Running a processor-intensive ImageOptim-task and noticed ST kinda lags, so I guess that caused it. Will fix...